src/nylas-connection.js

```diff
@@ -147,7 +147,7 @@ module.exports = class NylasConnection {
 
         if (error || response.statusCode > 299) {
           if (!error) {
-            error = new Error(body.message);
+            error = new Error(JSON.stringify(body)); 
```

Comment:
I updated this because the auth error code spits out the following response:

```
{
             "message": "Please include the missing fields",
             "type": "auth_error",
             "missing_fields": [
                 "provider"
             ],
             "error": "Missing fields."
}
```
If we only surface the error response's `body.message` value then it masks the error and isn't as helpful.
Not sure if it would have any negative affects on other endpoints...at the top of my head I don't see how it would, but would love to hear your thoughts!